spec/Section 7 -- Response.md

```diff
@@ -10,7 +10,7 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A response to a GraphQL request must be a map.
+A response to a GraphQL request must be a map or an event stream of maps.
```

Comment:
what is 'event stream'? why 'event' ?

Reply:
I think response stream would be correct ... event stream is for subscriptions.

Reply:
https://spec.graphql.org/June2018/#sec-Response-Stream

Reply:
In the server we used more the name source stream for event stream.

Reply:
https://spec.graphql.org/June2018/#sec-Source-Stream

Reply:
I will go over the spec a bit I think there is some ambiguity on  the terms

Reply:
I changed it to "response stream"